api.raml

```diff
@@ -0,0 +1,107 @@
+#%RAML 1.0
+title: ID Broker API
+protocols: [ HTTPS ]
+mediaType: application/json
+securitySchemes:
+  # TODO: Define auth. bearer token schema
+securedBy:
+  # TODO: Specify auth. bearer token schema
+types:
+  Authentication:
+    type: object
+    properties:
+      username: string
+      password: string
+  User:
+    type: object
+    properties:
+      employee_id: string
+      first_name: string
+      last_name: string
+      display_name?: string
+      username: string
+      email: string
+      active?: YesOrNo
+      locked?: YesOrNo
+  YesOrNo:
+    type: string
+    enum: [ yes, no ]
+/authentication:
+  post:
+    body:
+      application/json:
+        type: Authentication
+        example: |
+          {
+            "username": "john_smith",
+            "password": "correcthorsebatterystaple"
+          }
+    responses:
+      200:
+        description:
+          The given credentials were acceptable/valid (i.e. - the
+          authentication succeeded).
+        body:
+          application/json:
+            description: The attributes of the authenticated User.
+            type: User
+            example: |
+              {
+                "employee_id": "12345",
+                "first_name": "John",
+                "last_name": "Smith",
+                "display_name": "John Smith",
+                "username": "john_smith",
+                "email": "john_smith@example.com",
+                "active": "yes",
+                "locked": "no"
+              }
+      400:
+        description:
+          The given credentials were NOT acceptable/valid (i.e. - the
+          authentication failed).
+
```

Comment:
@fillup What kind of interface do we want to specify for this part of the conversation between the called (such as SilAuth) and ID Broker? Do we want/need to divulge whether the user didn't exist vs. a bad password was provided? SilAuth at this point doesn't care, but I don't know what would be the best design just from looking at the ID Broker by itself.